sample-hacep-project/sample-hacep-project-client/pom.xml

```diff
@@ -12,6 +12,10 @@
     <artifactId>sample-hacep-project-client</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>org.kie</groupId>
+            <artifactId>openshift-kie-hacep</artifactId>
```

Comment:
Why this new dependency? Client code should not depends on hacep impl

Reply:
Without this dep every API invocation to read a result lead to a class not found exception, is it time to move Command's result object to a separate module ?